src/save.ts

```diff
@@ -41,7 +41,9 @@ async function run(): Promise<void> {
         });
 
         try {
-            await cache.saveCache(cachePaths, primaryKey);
+            await cache.saveCache(cachePaths, primaryKey, {
+                uploadChunkSize: utils.getInputAsInt(Inputs.UploadChunkSize)
```

Comment:
`getInputAsInt` can return undefined, should we fail in that case or just pass in a default?

Reply:
This would use the default (32 MBs).  This is handled by [`getUploadOptions`](https://github.com/actions/toolkit/blob/main/packages/cache/src/options.ts#L56).  If undefined it fills in the default value.